Renewal/System/OngoingQuestInfoList_True_EN.lub

```diff
@@ -25173,20 +25276,20 @@ QuestInfoList = {
 		Summary = "Frozen Gargoyle, Prison Breaker Kill"
 	},
 	[8547] = {
-		Title = "A mirror-like thing",
+		Title = "A Mirror",
 		IconName = "ico_dq.bmp",
 		Description = {
-			"<NAVI>Nilem<INFO>glast_01,191,273,0,101,0</INFO></NAVI> asks you to kill 10 Ice Ghosts and 10 Flame Ghosts and see them change."
+			"<NAVI>[Mage Nillem]<INFO>glast_01,190,273,0,101,0</INFO></NAVI> asked you to hunt 10 Ice Ghosts and 10 Flame Ghosts to gather more information."
 		},
-		Summary = "Ice Ghost, Flame Ghost Kill"
+		Summary = "Hunt 10 Ice Ghosts and 10 Flame Ghosts."
 	},
 	[8548] = {
-		Title = "Mirror (Standby)",
+		Title = "A Mirror - Wait",
 		IconName = "ico_dq.bmp",
 		Description = {
-			"This quest is a daily quest, and can be performed by receiving <NAVI>Nilem<INFO>glast_01,191,273,0,101,0</INFO></NAVI> when the waiting time expires."
+			"This is a daily quest, you can take it again tomorrow when the timer expires. Speak with <NAVI>[Mage Nillem]<INFO>glast_01,190,273,0,101,0</INFO></NAVI>."
 		},
-		Summary = "Resets at 04:00",
+		Summary = "Reset at 04:00.",
```

Comment:
Why modify "Resets at 04:00"?
This is a phrase used thousands of times throughout the file.